sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -2292,11 +2297,11 @@ class AnalyzedDocument:
     :ivar doc_type: The type of document that was analyzed.
     :vartype doc_type: str
     :ivar bounding_regions: Bounding regions covering the document.
-    :vartype bounding_regions: list[~azure.ai.formrecognizer.BoundingRegion]
+    :vartype bounding_regions: Optional[list[~azure.ai.formrecognizer.BoundingRegion]]
     :ivar spans: The location of the document in the reading order concatenated content.
     :vartype spans: list[~azure.ai.formrecognizer.DocumentSpan]
     :ivar fields: A dictionary of named field values.
-    :vartype fields: dict[str, ~azure.ai.formrecognizer.DocumentField]
+    :vartype fields: Optional[dict[str, ~azure.ai.formrecognizer.DocumentField]]
```

Comment:
kind of nit picky, but if we're talking about Optional in a typing sense, it means it can be the value or None. I think in more cases than not, we return an empty dict for fields, right? Seeing as this is just the docstring, I think it's probably fine, though.

Reply:
I had the same thought when doing this. I'm going to bring this up to Anna and Johan to see about the correctness of returning the empty dict or list.

Reply:
If a change is needed I'll add it in a future PR.